src/components/AudioTrack/AudioTrack.vue

```diff
@@ -78,6 +82,8 @@ export default {
     player: null, // HTMLAudioElement
     currentTime: 0,
     duration: 0,
+    /** Amount in seconds to jump seek **/
+    seekJumpDuration: 15,
```

Comment:
I think this should be determined as a percentage of the track length, clamped between a minimum and maximum number of seconds. A lot of tracks from freesound are very short and I suppose audiobooks, if and when we add them, will be very long.

This is more of a UX question so maybe @panchovm can help us with the user perspective.

Reply:
Based on the behavior I've seen on other players, it should just jump one second when using the arrow keys. If we wanted to implement a special behavior for power users to seek over larger sections of time than one second we could add a "shift" modifier to accomplish that, but I think the default behavior should just be second by second (otherwise we're making it impossible for keyboard users to seek to a specific second).

Reply:
Agree with @sarayourfriend it sounds good exploring other options for power users and specific audio categories. I'm thinking of podcasts.

Reply:
I think let's do 1s, folks can hold down to do more, and we could do a shift modifier in the future that does a % of the track instead.

Reply:
could also always add dedicated `<- 30s` `30s ->` buttons to the player ui in the future!

Reply:
Currently set to 1s, 15s with modifier.